app/gateway/changelog.md

```diff
@@ -12,6 +12,11 @@ For product versions that have reached the end of sunset support, see the [chang
 ## 3.5.0.0
 **Release Date** TBD
 
+### Features
```

Comment:
❗ For reviewers: I think they said this will be backported into 2.8 and other supported versions, but I think the backport is happening in 2.8.4.5 (as marked on the KAG ticket), but that version isn't released yet and I'm not sure when it will be released and was wondering if anyone knew. I'm guessing we'll need to make a Jira ticket to back port this in our docs if it's not released at the same time as 3.5.